types/src/front/assistant/templates.ts

```diff
@@ -189,7 +196,10 @@ export function getAgentActionConfigurationType(
         dataSources: [],
         id: -1,
         tagsFilter: null,
-        relativeTimeFrame: "auto",
+        relativeTimeFrame: {
```

Comment:
Shouldn't this be `auto` if not specified since we need `auto` for retrieval actions generally?